### ISSUE TITLE
[FEATURE] Add event analytics endpoints

### ISSUE THREAD

**Issue by yoonlees** [rokwire/illinois-app#4451]
**Is your feature request related to a problem? Please describe.**
There are no issues with the current Illinois app analytics APIs, but we would like to add more.

**Describe the solution you'd like**
We are planning to create multiple tickets related to events APIs. First, we want to consume APIs to retrieve data for the calendar database. We aim to utilize the information from the events and events_attendances collection fields to answer the following question:
- For how many events was attendance taken?

**Comment by mihail-varbanov** [comment]
I think the Illinois client app should not be involved in this endpoint. Splunk service should access Calendar BB directly instead and obtain the consolidated event data that they are interested in. In general, the client app logs the user's activity. The requested API is not related to any user or to any activity, so the model should be totally different than what we currently have as a relation  between the Illinois client app and Splunk logging service.

**Comment by kpecknold** [comment]
That makes sense to me. Should these requests go in a different repo?

**Comment by mihail-varbanov** [comment]
Yes, that's right. These should go to Calendar BB's [repo](https://github.com/rokwire/calendar-building-block).
